code/_onclick/hud/screen_objects.dm

```diff
@@ -126,13 +126,16 @@
 		if(!C.incapacitated())
 			if(C.internal)
 				C.internal = null
-				C << "<span class='notice'>No longer running on internals.</span>"
+				C << "<span class='notice'>You are no longer running on internals.</span>"
 				icon_state = "internal0"
 			else
 				if(!istype(C.wear_mask, /obj/item/clothing/mask))
-					C << "<span class='notice'>You are not wearing a mask.</span>"
+					C << "<span class='warning'>You are not wearing a mask!</span>"
 					return 1
 				else
+					var/obj/item/clothing/mask/M = C.wear_mask

```

Comment:
I tried that, but now it says "You need an internals mask blahblahblah !" when I try to turn internals on with the breath mask adjusted down.